libflux/src/flux/semantic/walk/walk_mut.rs

```diff
@@ -229,7 +229,7 @@ impl<'a> NodeMut<'a> {
             NodeMut::Block(Block::Expr(ref mut estmt, _)) => {
                 mut_expr_loc(&mut estmt.expression, loc)
             }
-            NodeMut::Block(Block::Return(ref mut expr)) => mut_expr_loc(expr, loc),
+            NodeMut::Block(Block::Return(ref mut expr)) => mut_expr_loc(&mut expr.argument, loc),
```

Comment:
Unless I'm missing something, I don't think we need to do anything here for any of the block variants.  The `Block` enum has a `loc()` method that just produces the location of it's statement, and the statement will be visited elsewhere in this enum.

So I think for Block this should be:
```rust
NodeMut::Block(_) => (),
```

Reply:
Yeah, that does actually make sense. I'll update that.